### DISCUSSION TITLE
A wild discussion turns to an issue

### DISCUSSION THREAD

**Discussion by snowsky** [apache/airflow#40734] [General]
### What happened?

I don't mind to challenge @potiuk's authority if I am not making a mistake, https://github.com/apache/airflow/discussions/40590. What I need, and I think this may also help others, to make sure Airflow is not impacted by Polyfill. Sorry, I'm stubborn when I want to find an answer.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
You can derive answer from the discussion

**Comment by snowsky** [comment]
> You can derive answer from the discussion

Haha, you do have powers. Let us be technical, deriving is not always technical.

**Comment by snowsky** [comment]
According to https://www.apache.org/foundation/policies/conduct.html, I think @potiuk is not 
- Be open
- Be [empathetic](https://www.apache.org/foundation/policies/conduct#endnotes), welcoming, friendly, and patient
- Be collaborative
- Be inquisitive
- Be careful in the words that we choose
- Be concise

But only fit the last one:
- Step down considerately

**Comment by potiuk** [comment]
Yes you are stubborn. I propose you apply those points to your situation:

* be open - to the fact that sometimes when you get software for free, it's up to you to investigate and if you have an easy * way to check if there is a dependency used, you can check it
* be empathetic to the maintainers here. Asking security related question will raise their hair (if they still have them - I don't) and doing it unnecessarily (as explained multiple time but you seem to be ignoring it every single time I mention it) is a noise that drags then out from other issues. Every day there are many 10s of high/critical issues. If we have every user of ours asking about security issues in unrelated dependencies that we DON'T have and expect an answer, we would stop developing Airflow as all our time would be spend on answering
* be collaborative - I asked you several times to collaborate and double check if the dependency you mentioned is even used by Airflow and gave you means to do so - so that you cand find answer yourself with the spirit of:
* be inqusitive - and learn how to check it next time without bothering anyone
* be careful with the words you chosoe and concise - maybe

I propose you:

* step down considerably

Also - for the inquisitive part: I propose you watch teh talk I gave at Airflow summit about empathy https://airflowsummit.org/sessions/2022/hey-maintainer-exercise-your-empathy/  - where I try to explain that both maintainers and users should exercise more empathy. You should watch it and try to put yourself in maintainer shoes and see why rather than giving direct answer I did what I dit - mainly so that you (and 100s of others) are not coming next time and aksing similar questions about unrelated security issues but go and check on their own using the means we already have.

You might also look at the stats of the discussion and find out that I am consistently most helpful person there (by far) who answers and helps our users a lot when they need help. But I also think it's worth to teach people like you how they can make use of the information they have rather than introduce noise.

**Comment by snowsky** [comment]
Again I don't think what you said is on behalf of Airflow or Apache Foundation. It is your point of view. From the video you said you rejected requests very frankly. This is against `Be Open` policy. You are shutting down many potential good developers. If you promote this in your video but didn't mention it doesn't relate to the organization as a disclaimer, you are blurring the boundary between self view and organization vision. The organization should not publish your video to the public as well.

Next time, please don't repeat others view so you will not be labeled as suspicious Plagiarism. I'm tired of talking to you as many others that you don't see what I'm seeing. But if you'd like I can show what I have seen, and at the same time, I can listen to you because I'm more `Be Open`. Lol

**Comment by snowsky** [comment]
Don't pretend to be helpful by shutting down people! This is a psycho game but I don't buy it.

**Comment by snowsky** [comment]
If you label yourself `Be Open`, can you reopen the issue to let others join the discussion?

**Comment by potiuk** [comment]
No but I am open to convert it into discussion and see if anyone would be willing to comment on it.

**Reply by snowsky** [reply]
This is interesting, there was an existing discussion, you can reopen that one instead of suspending it. No sense to convert an issue to discussion again. You management ability is high doubted.

**Reply by snowsky** [reply]
I guess you are not sure if Airflow is impacted or not so you want the community inputs. `Be concise` code of conduct is violated!

**Reply by potiuk** [reply]
Surely I can reopen the original discussion. Hopefully someone will be able to help you. I am closing his one instead to avoid duplicates.

**Reply by snowsky** [reply]
You are not opening the old one, which is still locked but only open to collaborators. We are cycling back to the original point...

**Comment by snowsky** [comment]
There is no point converting an issue to an existing discussion, https://github.com/apache/airflow/discussions/40590#discussioncomment-9960660. Can anyone else other than @potiuk take a look? This is becoming a harassment.
